pos_bahrain/pos_bahrain/doctype/pos_closing_voucher/pos_closing_voucher.py

```diff
@@ -144,10 +166,45 @@ def get_form_collected(mop):
                     make_payment(payment), get_form_collected(payment.mode_of_payment)
                 ),
             )
+        for payment in collection_payments:
+            self.append(
+                "payments",
+                merge(
+                    make_payment(payment), get_form_collected(payment.mode_of_payment)
+                ),
+            )
+
```

Comment:
Won't this create two rows of the same **Mode of Payment**, if this **Mode of Payment** happens to be present in both `actual_payments` and `collection_payments`? Or is this how the specs were?

Reply:
I think you are correct. Will check it. Thanks for pointing out!